doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -0,0 +1,947 @@
+2023_10_02_18_10_00     reboot                      Sun 02 Oct 2023 06:10:00 PM UTC     admin   User issued 'reboot'
+```
+#### 3.4.3 Reboot Cause CLIs on the Switch      <font>**`Executed on the switch`**</font>
+* The "show reboot-cause" CLI on the switch shows the most recent rebooted device, time and the cause. The could be the NPU or any DPU
+* The "show reboot-cause history" CLI on the switch shows the history of the Switch and all DPUs
+* The "show reboot-cause history module-name" CLI on the switch shows the history of the specified module
+
+"show reboot-cause history"      <font>**`Executed on the switch`**</font>
+```
+root@sonic:~#show reboot-cause
+
+Device          Name                    Cause                       Time                                User    Comment
+
+switch          2023_10_20_18_52_28     Watchdog:1 expired;         Wed 20 Oct 2023 06:52:28 PM UTC     N/A     N/A
+
+root@sonic:~#show reboot-cause history
+
+Device          Name                    Cause                       Time                                User    Comment
+
+switch          2023_10_20_18_52_28     Watchdog:1 expired;         Wed 20 Oct 2023 06:52:28 PM UTC     N/A     N/A
+switch          2023_10_05_18_23_46     reboot                      Wed 05 Oct 2023 06:23:46 PM UTC     user    N/A
+DPU0            2023_10_04_18_23_46     Power Loss                  Tue 04 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_03_18_23_46     Watchdog: stage 1 expired;  Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_02_18_23_46     Host Power-cycle            Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3            2023_10_02_17_23_46     Host Reset DPU              Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+DPU2            2023_10_02_17_20_46     reboot                      Sun 02 Oct 2023 05:20:46 PM UTC     admin   User issued 'reboot'
+
+"show reboot-cause history <module-name>"
+
+root@sonic:~#show reboot-cause history dpu3
+
+Device      Name                    Cause                           Time                                User    Comment 
+   
+DPU3        2023_10_03_18_23_46     Watchdog: stage 1 expired;      Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3        2023_10_02_18_23_46     Host Power-cycle                Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3        2023_10_02_17_23_46     Host Reset DPU                  Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+```
+#### 3.4.4 Chassis Module Status
+* The "show chassis modules status" is an existing CLI but extended to include the status of all DPUs and switch. <font>**`Executed on the switch. This CLI is not available on the DPU.`**</font>
+```
+root@sonic:~#show chassis modules status                                                                                      
+Name        Description         ID         Oper-Status     Admin-Status     Serial
```

Comment:
Why must the 'Serial' column be displayed in this command?

Reply:
This is an existing common CLI and the serial is there.  So don't want to make any changes to it.